sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVOptions.scala

```diff
@@ -90,6 +90,7 @@ class CSVOptions(
   val quote = getChar("quote", '\"')
   val escape = getChar("escape", '\\')
   val comment = getChar("comment", '\u0000')
+  val lineSeparator = parameters.getOrElse("lineSeparator", "\n")
```

Comment:
Ah, it was `wholeFile` but just renamed https://github.com/apache/spark/commit/2051428173d8cd548702eb1a2e1c1ca76b8f2fd5 to `multiLine` four hours ago.

Yes, it would be wonderful if we can support it in reading too. If it is difficult, it is fine (to me) to turn this PR for hard corded one.